zerorobot/robot/robot.py

```diff
@@ -111,6 +109,7 @@ def start(self,
         logger.info("data directory: %s" % config.data_repo.path)
         logger.info("config directory: %s" % j.tools.configmanager.path)
         logger.info("sshkey used: %s" % os.path.expanduser(os.path.join('~/.ssh', j.tools.configmanager.keyname)))
+        logger.info("configmanager sshkeypath: "+ j.tools.configmanager.keypath)
```

Comment:
this is exactly the same as the line before. Remove it

Reply:
sandbox might contain the keys, and will always contains the keys in case of zdb backend so the full path of the key in case of zdb backend is the right thing to show